src/lcm/simulate.py

```diff
@@ -504,83 +490,20 @@ def create_data_scs(
 
     # get sparse and dense choices
     # ==================================================================================
-    sparse_choices = {
-        name: grid
-        for name, grid in model.grids.items()
-        if name in vi.query("is_sparse & is_choice").index.tolist()
-    }
-
     dense_choices = {
         name: grid
         for name, grid in model.grids.items()
-        if name in vi.query("is_dense & is_choice & ~is_continuous").index.tolist()
+        if name in vi.query("is_choice & ~is_continuous").index.tolist()
```

Comment:
```suggestion
        if name in vi.query("is_choice & is_discrete").index.tolist()
```
Much easier to read.